.eslintrc.json

```diff
@@ -1,4 +1,3 @@
-// Default eslintrc for packages without one, or files outside a package
 {
```

Comment:
## [🕵 FluentUIV0 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28583&commitId=9f4b88e77690bb168f44fce641ef19d6a1bd11c5&env=prod&clientType=FLUENTUI)


  <details><summary><b>Popup</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Popup.Popper Example Visibility Modifiers.has  [data-popper-is-intersecting]  when the popover intersects boundaries.chromium.png|1496| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v0"/>